src/main/java/org/storageart/storageart/service/UserService.java

```diff
@@ -0,0 +1,21 @@
+package org.storageart.storageart.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+import org.storageart.storageart.repositories.UserRepository;
+
+@Service
+public class UserService implements UserDetailsService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException{
+        return userRepository.findByUsername(username).get();
```

Comment:
You should throw UsernameNotFoundExcpetion if there is no user with given username, not just return null